pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -41,6 +41,7 @@ private CommonConstants() {
   public static final String DEFAULT_METRICS_FACTORY_CLASS_NAME =
       "org.apache.pinot.plugin.metrics.yammer.YammerMetricsFactory";
 
+  public static final String DEFAULT_SWAGGER_AUTHORIZATION_VALUE = "oauth";
```

Comment:
maybe change the name to `SWAGGER_AUTHORIZATION_KEY` - and reuse it in both, the `@ApiKeyAuthDefinition` and the `@Authorization` annotations rather then the hardcoded `"oauth"` string?

Reply:
good point. Changed now